src/scan/postgres_scan.cpp

```diff
@@ -201,7 +205,7 @@ PostgresReplacementScan(duckdb::ClientContext &context, duckdb::ReplacementScanI
 	}
 
 	/* SELECT query will have nodePath so we can return cardinality estimate of scan */
-	Cardinality nodeCardinality = node_path ? node_path->rows : 1;
```

Comment:
I don't think this is needed for PG15 support (probably this was for PG14 support). So let's remove it for now. Also it's the wrong type afaict. The Cardinality is a typedef for `double`.

Reply:
Ok, here the type definition basis function "CreateFunctionIndexScanArguments" parameter
![image](https://github.com/user-attachments/assets/ef0cf67d-97a8-41c2-ba91-fb461eed73ed)

Reply:
Hmm, I guess Postgres and DuckDB cardinalities have a different type (uint64 for Duckdb and double for Postgres). Let's leave it as is for now since this only applies to PG14, but let's add a comment explaining this difference on the PG14 PR.